schema/me/lot_standings.js

```diff
@@ -0,0 +1,47 @@
+import gravity from '../../lib/loaders/gravity';
+import LotStanding from './lot_standing';
```

Comment:
Also not a big deal but you can just get the type directly without doing `LotStanding.type`:
```js
import { LotStandingType } from './lot_standing';
```